package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commander",
-  "version": "2.9.0",
+  "version": "2.9.1",
```

Comment:
Shouldn't we increase minor version instead of patch, because of "Allow literal '--' to be passed-through as an argument" feature?